modules/Bio/Otter/Lace/Client.pm

```diff
@@ -966,21 +967,21 @@ sub _find_clone_result {
 
 sub get_meta {
     my ($self, $dsname) = @_;
-    my $hashref = $self->otter_response_content(GET => 'get_meta', { dataset => $dsname });
+    my $hashref = $self->otter_response_content(GET => 'get_meta', { dataset => $dsname, 'author' => $self->author });
```

Comment:
We always transfer the same author to this function. It would be better set default author once inside the function, since function is placed in this script also

Reply:
Every script requires author information. There is no default author. It changes with every user.